python/openchange/provision.py

```diff
@@ -621,12 +638,16 @@ def newuser(names, lp, creds, username=None, mail=None):
 add: legacyExchangeDN
 legacyExchangeDN: /o=%(firstorg)s/ou=%(firstou)s/cn=Recipients/cn=%(username)s
 add: proxyAddresses
+proxyAddresses: SMTP:%(smtp_user)s
```

Comment:
In which specification do you have this order thing? I have none.

The SMTP indicates the PRIMARY proxy address to use and the smtp ones are the SECONDARIES. Note the case. See [MS-OXCMAIL] Section 1.1 for details.

I will revert this change and provide the migration to the current status in the `directory app`. About the order, no problem on keeping it as it is if no more regressions are found.